### PR TITLE
docs: fix article "Add a File" to guide the user to click resource name and to use the tabs

### LINKED ISSUE #861: [docs]: fix article "Add a File"
### Description

Article [Add a File](https://www.devopness.com/docs/files/add-file/) needs to be updated, more details @ #855 

### Issue List

- [ ] Guide the user to click Resource Name
  - Steps below needs to be fixed accordingly
  ```
  1. In the list of applications find the application where you want to add a configuration file and click DETAILS
  ```
- [ ] Guide the user to use Tabs
  - Steps below needs to be fixed accordingly
  ```
  1. On the upper-right corner of the applications details view click SETTINGS
  1. Use the drop-down menu to choose CONFIGURATION FILE
  1. On the right corner of the list click ADD CONFIGURATION FILE
  ```

### PR DESCRIPTION
**Pull request by dudunog**
## Description of changes

- [x] Fixed the excerpt "In the list of applications find the application where you want to add a configuration file and click DETAILS" that was replaced by a more descriptive text explaining how to enter in the file details view.
- [x] Fixed the excerpts that guide the user to add a configuration file to use the tabs

## GitHub issues resolved by this PR

- [x] This PR resolves #861 

## Quality Assurance

- Once the changes in this PR are merged and deployed, success criteria is: Update the article https://www.devopness.com/docs/files/add-file/

### CODE REVIEW

**Review by jfoliveira — APPROVED** [review]

File: docs/files/add-file.md [hunk 1]
```diff
@@ -16,10 +16,9 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
-1. In the list of applications find the application where you want to add a configuration file and click `DETAILS`
-1. On the upper-right corner of the applications details view click `SETTINGS`
-1. Use the drop-down menu to choose `CONFIGURATION FILE`
-1. On the right corner of the list click `ADD CONFIGURATION FILE`
+1. In the list of applications, find the application where you want to add a configuration and click the `NAME` of the application
+1. Click the `CONFIGURATION FILES` tab
+1. On the right corner of the applications tab click `MANAGE`
```
Comment (thlmenezes) [inline docs/files/add-file.md:21]: In cases like this other articles, as [Add Pipeline](https://www.devopness.com/docs/pipelines/add-pipeline/) [1], use the tab name

```suggestion
1. On the right corner of the configuration files tab click `MANAGE`
```

[1]

https://github.com/devopness/devopness/blob/1226718ee38cf9d5ab3ee64e85d334642468e938/docs/pipelines/add-pipeline.md?plain=1#L21

**Review by thlmenezes — CHANGES_REQUESTED** [review]

File: docs/files/add-file.md (hunk 1 above)
Comment (dudunog) [reply docs/files/add-file.md]: Ok, fixed!

**Review by thlmenezes — DISMISSED** [review]

File: docs/files/add-file.md [hunk 2]
```diff
@@ -16,10 +16,9 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
-1. In the list of applications find the application where you want to add a configuration file and click `DETAILS`
-1. On the upper-right corner of the applications details view click `SETTINGS`
-1. Use the drop-down menu to choose `CONFIGURATION FILE`
-1. On the right corner of the list click `ADD CONFIGURATION FILE`
+1. In the list of applications, find the application where you want to add a configuration and click the `NAME` of the application
```
Comment (jfoliveira) [inline docs/files/add-file.md:19]:
```suggestion
1. In the list of applications, find the application to which you want to add a configuration and click the `NAME` of the application
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/files/add-file.md (hunk 2 above)
Comment (dudunog) [reply docs/files/add-file.md]: Fixed!

**Review by thlmenezes — APPROVED** [review]

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**